history-of-the-quakers-v3/original/16-book-5-chapter-1.adoc

```diff
@@ -130,8 +142,11 @@ from prisons, spoil and ruin.
 And we shall, as in christian duty bound, pray God for the king`'s welfare in this world,
 and his eternal happiness in that which is to come.
 
+[.signed-section-context-close]
 London, the 2nd of the 1st month called March, 1684-5.
 
+--
+
 Soon after the friends of London drew up a more copious state of their case and sufferings,
 addressed to the king and both houses of parliament,
 wherein they recapitulate their grievous sufferings
```

Comment:
I can't leave a note on line 157, but a footnote starts there listing some things and there are paragraph breaks between each one, and I wasn't sure if I should leave them or delete them like that other footnote that had discourse in it.